lib/tasks/notify.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :notify do
+  desc "Send an email notification"
+  task :send_email, [:email_address] => :environment do |_, args|
+    raise "Missing email address" unless args.email_address
+
+    headers = {
+      to: args.email_address,
+      template_id: ENV["TEMPLATE_ID"] || "759acac6-da53-4a19-b591-b7538c7c39de",
```

Comment:
Not a concern for now but do we need to set this key up to be an environment variable (ignoring the one as an argument) so we can have different ones in integration/staging/prod

Reply:
Sure, once we've got the actual template ids then I can delete this one.